src/DynamoCore/Configuration/PreferenceSettings.cs

```diff
@@ -322,7 +322,7 @@ public double DefaultScaleFactor
         /// <summary>
         /// Indicates whether background preview is active or not.
         /// </summary>
-        [Obsolete("Property will be deprecated in Dynamo 3.0, please use BackgroundPreviews")]
+        [Obsolete("Property will be deprecated in a future version of Dynamo, please use BackgroundPreviews")]
```

Comment:
how come this was not removed?

Reply:
I got too complicated (too much refactoring).

Reply:
(At least for now)

Reply:
internal

Reply:
I found a way.